src/index.html

```diff
@@ -1,17 +1,143 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+      content="width=device-width, user-scalable=no,
+                initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+    >
+      <fieldset class="field-group">
+        <legend>Personal Information</legend>
+        <label class="form-field">Surname:
+          <input
+            name="surname"
+            type="text"
+            autocomplete="off"
+          >
```

Comment:
this is a proper way to format your code, pay attention to such things
```suggestion
        <label class="form-field">
          Surname:
          <input
            name="surname"
            type="text"
            autocomplete="off"
          >
```